sse2neon.h

```diff
@@ -1045,7 +1052,7 @@ FORCE_INLINE __m128 _mm_shuffle_ps_default(__m128 a,
         }                                                  \
         ret;                                               \
     })
-#endif  // not clang
+#endif  // no __builtin_shufflevector
```

Comment:
I think the comment `no __builtin_shufflevector` should be removed since the `#endif` just means that it is the end of the `#if` and `#else` combination.